src/base.yml

```diff
@@ -9,5 +9,4 @@
     - dev_ssh_access
     - htop
     - persist_journald
-    - xfce
-    # - xfce_cool
+    - xfce_cool
```

Comment:
Do we still need to explicitly require `xfce` in `src/requirements.yml`?

Reply:
Good point.  See commit 4db81bd.